builtin/providers/aws/resource_aws_config_config_rule.go

```diff
@@ -110,6 +110,7 @@ func resourceAwsConfigConfigRule() *schema.Resource {
 									"event_source": {
 										Type:     schema.TypeString,
 										Optional: true,
+										Default:  "aws.config",
```

Comment:
I'm not sure we need to. If someone has a `source_detail` in their elb then they have to have `aws.config` or it'll error out like in #14074. We could do a migration just to be safe but I don't think it's needed